Makefile

```diff
@@ -78,6 +78,12 @@ reset-site-database:
 shell: ## Get a shell inside the Drupal container
 	docker compose exec drupal bash
 
+ut: update-translations ## Update Drupal from local .po translation files
+update-translations:
+	docker compose exec drupal drush locale:clear-status
+	docker compose exec drupal drush locale:update
+	docker compose exec drupal drush cache:rebuild
```

Comment:
will we want to add any of these steps to our post deploy script?

Reply:
Oops! Yeah definitely, maybe that will help with my issues here...